src/components/interior-left-nav/_interior-left-nav.scss

```diff
@@ -209,12 +209,12 @@
     }
 
     .left-nav-list__item--expanded > .left-nav-list__item-link {
-      color: #152935;
+      color: $text-01;
     }
 
     .left-nav-line {
       margin: rem(24px) rem(20px);
-      border: 1px solid #0f212e;
+      border: 1px solid darken($active-primary, 10);
```

Comment:
- [ ] Change to hex and override with `body.bx--v9-theme`  or find a token due to the css custom properties issue or other solution.